vendor/github.com/syndtr/goleveldb/leveldb/errors/errors.go

```diff
@@ -64,7 +64,10 @@ type ErrMissingFiles struct {
 	Fds []storage.FileDesc
 }
 
-func (e *ErrMissingFiles) Error() string { return "file missing" }
+func (e *ErrMissingFiles) Error() string {
+	msg := fmt.Sprintf("file missing : %s ", e.Fds)
+	return msg
+}
```

Comment:
Editing a vendored file in this way is unfortunately not okay. We've no proper means to ensure that change sticks around. You'd have to upstream it first.